run_F2010SC5-CMIP6_E3SMv2_alpha5_59_wP3_cori.csh

```diff
@@ -0,0 +1,1622 @@
+#  set input_data_dir = '/project/projectdirs/m411/ACME_inputdata'    # PJC-NERSC
+## set input_data_dir = '/project/projectdirs/ccsm1/inputdata'        # NERSC
+#else if ( $machine == 'titan' || $machine == 'eos' ) then
+#  set input_data_dir = '/lustre/atlas/proj-shared/cli112/pjcs/ACME_inputdata'    # PJC-OLCF
+#endif
+#if ( -d  $input_data_dir ) then
+#  $xmlchange_exe --id DIN_LOC_ROOT --val $input_data_dir
+#else
+#  echo 'run_e3sm ERROR: User specified input data directory does NOT exist.'
+#  echo '                $input_data_dir = '$input_data_dir
+#  exit 270
+#endif
+
+### The following command extracts and stores the input_data_dir in case it is needed for user edits to the namelist later.
+### NOTE: The following line may be necessary if the $input_data_dir is not set above, and hence defaults to the E3SM default.
+#NOTE: the following line can fail for old versions of the code (like v0.3) because
+#"-value" is a new option in xmlquery. If that happens, comment out the next line and
+#hardcode in the appropriate DIN_LOC_ROOT value for your machine.
+set input_data_dir = `$xmlquery_exe DIN_LOC_ROOT --value`
+
+#============================================
+# COMPONENT CONFIGURATION OPTIONS
+#============================================
+
+#NOTE:  This section is for making specific component configuration selections.
+#NOTE:  The input_data directory is best set in the section for it above.
+#NOTE:  Setting CAM_CONFIG_OPTS will REPLACE anything set by the build system.
+#       To add on instead, add '-append' to the xmlchange command.
+#NOTE:  CAM_NAMELIST_OPTS should NOT be used.  Instead, use the user_nl section after case_scripts.build
+
+# +++ 
+$xmlchange_exe --id CAM_CONFIG_OPTS --val "-mach $machine -phys cam5 -clubb_sgs -microphys p3 -chem linoz_mam4_resus_mom_soag -rain_evap_to_coarse_aero -bc_dep_to_snow_updates -nlev 72 -cosp"
+# +++
+
+# Use atmosphere theta-l dynamical core
+${xmlchange_exe} CAM_TARGET=theta-l
+
+${xmlchange_exe} CAM_CONFIG_OPTS="-cosp" --append
+
+# Switch radiation to RRTMGP
+## [JS]:comment-out   ${xmlchange_exe} --append CAM_CONFIG_OPTS='-rad rrtmgp'
+
+# Build with COSP
+# NOTE: The xmlchange will fail if CAM is not active, so test whether a data atmosphere (datm) is used.
+if ( `$xmlquery_exe --value COMP_ATM` == 'datm'  ) then 
+  e3sm_newline
+  e3sm_print 'The specified configuration uses a data atmosphere, so cannot activate COSP simulator.'
+  e3sm_newline
+else
+  e3sm_newline
+  e3sm_print 'Configuring E3SM to use the COSP simulator.'
+  e3sm_newline
+  $xmlchange_exe --id CAM_CONFIG_OPTS --append --val='-cosp'
+endif
+
+
+#===========================
+# SET THE PARTITION OF NODES
+#===========================
+
+if ( `lowercase $debug_queue` == true ) then
+  if ( $machine == quartz || $machine == syrah ) then
+    $xmlchange_exe --id JOB_QUEUE --val 'pdebug'
+  else if ($machine != sandiatoss3 && $machine != bebop && $machine != blues) then
+    $xmlchange_exe --id JOB_QUEUE --val 'short'
```

Comment:
@crterai 
Here I think the `short` queue on Cori is named 'debug'

Reply:
agreed. I've updated this now